src/app/components/Typography.ts

```diff
@@ -0,0 +1,13 @@
+const fontsize = {
+  h3: '2.4rem',
+};
```

Comment:
in here you can 

```typescript
export const h3Style = css`
  font-size: ...;
  font-weight: bold;
  color: ...;
  line-height: ...;
`;
export const H3 = styled.h3`
  ${h3Style}
`;
```

and then in the navbar component you can extend those styles with whatever overrides (like the color) like this

```typescript
const CoolHeading = styled(H3)`
  color: ...;
  padding: ...;
`;
```

and in the main nav, where you have the login text that uses the h4 styles but is not actually an h4 because it needs to be an `a`, you can do

```typescript
const LoginLink = styled.a`
  ${h4Style}
  
  // hover state
`;